internal/cloud/realm/access_list_test.go

```diff
@@ -29,6 +29,12 @@ func TestRealmIPAccess(t *testing.T) {
 		testApp, teardown := setupTestApp(t, client, groupID, "accesslist-test")
 		defer teardown()
 
+		t.Run("should have no allowed IPs upon app initialization", func(t *testing.T) {
+			accessList, err := client.AllowedIPs(groupID, testApp.ID)
+			assert.Nil(t, err)
+			assert.Equal(t, 0, len(accessList.AllowedIPs))
```

Comment:
worth having an assert on the `current_ip` here as well?

Reply:
Since we aren't returning the `current_ip` anymore, we can simply ignore it.